keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -1656,30 +1662,26 @@ gets updated now if the field doesn't
 match the current list already. If no PodScheduling object exists yet,
 it gets created.
 
-When there is a scheduler extender configured for the claim, creating and
-updating PodScheduling objects gets skipped because the scheduler
-extender handles filtering.
-
 #### Reserve
 
 A node has been chosen for the Pod.
 
 If using delayed allocation and the resource has not been allocated yet,
-the SelectedNode field of the ResourceClaim
+the PodSchedulingSpec.SelectedNode field
```

Comment:
You could do a single API call with the data for PotentialNodes and SelectedNode.

You could store the necessary information in memory in PreScore.

Reply:
I think the implementation already does that. Let me double-check and update the KEP update accordingly.

Reply:
Done.